shell/platform/darwin/macos/framework/Source/FlutterView.mm

```diff
@@ -15,38 +15,37 @@ @interface FlutterView () <FlutterResizeSynchronizerDelegate> {
   __weak id<FlutterViewReshapeListener> _reshapeListener;
   FlutterResizeSynchronizer* _resizeSynchronizer;
   FlutterSurfaceManager* _surfaceManager;
+  NSOpenGLContext* _openGLContext;
```

Comment:
I'm probably missing something but what's the main purpose of this refactor if FlutterView still owns the `openGLContext`? @iskakaushik